data/.gitignore

```diff
@@ -1 +1,21 @@
 regression
+
+# ORCA
```

Comment:
I meant that maybe we don't need all these ignores at all. I imagine the types of temporary files can change over time, and these ignores are specific to your local setup where you run calcs in this directory. Could you keep the ignores local for these reasons?